app/models/feature.rb

```diff
@@ -1,12 +1,12 @@
 class Feature
-  def self.enabled?(feature_name)
+  def self.enabled?(feature_name, user: nil)
     case feature_name.to_sym
     when :new_developer_fields_banner
       true
     when :pricing_v2
       true
     when :role_level_filters
-      !Rails.env.production?
+      !Rails.env.production? || user&.admin?
```

Comment:
```suggestion
      !Rails.env.production? || user.admin?
```

After requiring `user`, from the comment above.